packages/dashboard-core-plugins/src/panels/ConsolePanel.tsx

```diff
@@ -143,19 +207,21 @@ export class ConsolePanel extends PureComponent {
     return id;
   }
 
-  handleTabBlur() {
-    if (this.consoleRef) {
-      this.consoleRef.blur();
+  handleTabBlur(): void {
+    if (this.consoleRef && this.consoleRef.current) {
+      ((this.consoleRef.current as unknown) as HTMLElement).blur();
```

Comment:
This doesn't make sense - the ref is for `Console`, which is not an `HTMLElement`. It does not have a `blur()` function.
I think just remove this entire `handleTabBlur` method and all refs to it in `ConsolePanel`. I believe we had a use for this on Enterprise but it's not in Community and it simply wasn't wired up.